contracts/QuestFactory.sol

```diff
@@ -33,7 +31,7 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
     address public claimSignerAddress;
     address public protocolFeeRecipient;
     address public erc20QuestAddress;
-    address public erc1155QuestAddress;
+    address public erc1155QuestAddress; // todo remove this
```

Comment:
Curious, how will it be possible to remove this? Do you just assign a new address variable that could overwrite this slot?